org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/javadoc/JavadocContentAccess2.java

```diff
@@ -1946,6 +1947,7 @@ private void handleParameterTags(List<TagElement> tags, List<String> parameterNa
 				if (description != null) {
 					fBuf.append(description);
 				}
+				fBuf.append(BlOCK_TAG_ENTRY_END);
 				fBuf.append(BLOCK_TAG_END);
 			}
 		}
```

Comment:
`<li><b>Parameters:</b><ul><li><b>i</b>  an int</li></ul>`.
This is the doc appended by this `handleParameterTags(...)` function. 

I believe it misses an ending `</li>` tag, you should add `fBuf.append(BlOCK_TAG_ENTRY_END);` to the last line of this function.

Reply:
`handleExceptionTags(...)` also misses an ending `</li>` tag. Because it uses `handleBlockTagTitle(...);` at the beginning but misses a `</li> tag` at the ending.

Reply:
You're right. Looking at the other places where handleBlockTagTitle(..) is called makes it clear the ending entry needs to be added manually. Interesting piece of code.